Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.0.2'
+ruby '3.0.0'
```

Comment:
is it make sense to change the ruby version?

Reply:
When I tried to install gems for a project it said me that we have a mismatch in `.ruby-version` and `Gemfile`'s Ruby version

So, which one is the correct one?

Reply:
At the moment we are using version 3.0.2, but it's not a problem to upgrade to another version

Reply:
If 3.0.2 is the version that you use, I'll make it another way: 
I'll change `.ruby-version` to 3.0.2

Reply:
Thanks a lot